src/Context/Cart/CalculatedTax.php

```diff
@@ -27,6 +27,12 @@ public function getTax(): float
         return $this->data['tax'];
     }
 
+    public function getLabel(): ?string
+    {
+        \assert(is_string($this->data['label']) || is_null($this->data['label']));
+        return $this->data['label'] ?? null;
```

Comment:
`?? null` has no effect when assertions are enabled, as the assertion requires `label` to be set anyway.